### PR TITLE
fix intensity measure on vulnerability details page / vulnerability details page currently doesn't load

### PR DESCRIPTION
**Pull request by ragkoushik**
Revert template literal

### CODE REVIEW

File: assets/js/dataDetails.js [hunk 1]
```diff
@@ -83,14 +83,18 @@ $(document).ready(function() {
...
           case "intensity_measure":
-            let intensity_concat = [];
-            if(dataset[key] && dataset[key].length > 0){
-              dataset[key].forEach(function(intensity, idx){
-                intensity_concat.push(`${intensity} (${dataset['unit'][idx]})` )
-              })
+            if(Array.isArray(dataset[key])){
+              let intensity_concat = [];
+              if(dataset[key] && dataset[key].length > 0){
+                dataset[key].forEach(function(intensity, idx){
```
Comment (liaa2) [inline assets/js/dataDetails.js:89]: There is an error message when running it locally - after clicked vulnerability data? Do you have this error locally?
Screen Shot 2020-09-29 at 2 01 21 pm

**Review by liaa2 — CHANGES_REQUESTED** [review]

File: assets/js/dataDetails.js (hunk 1 above)
Comment (ragkoushik) [reply assets/js/dataDetails.js:89]: This is what I believe is fixed. Can you clear your cache and reload your browser and let me know if you still have issues?

Comment (liaa2) [reply assets/js/dataDetails.js:89]: Still exists, let me rebuild it

Comment (liaa2) [reply assets/js/dataDetails.js:89]: all good👍

**Review by liaa2 — APPROVED** [review]

**Pull request merged**